### PR TITLE
Add unicode support

### PR DESCRIPTION
**Pull request by mkassner**

### CODE REVIEW

File: pyglui/ui_elements.pxi [hunk 1]
```diff
@@ -5,7 +5,7 @@ cdef class UI_element:
     '''
     The base class for all UI elements.
     '''
-    cdef readonly bytes label
+    cdef readonly object label
```
Comment (papr) [inline pyglui/ui_elements.pxi]: Why do you use `object` in this case and not `str`? Might be more consistent with the rest of the file.

File: pyglui/ui_elements.pxi [hunk 2]
```diff
@@ -605,6 +611,14 @@ cdef class Text_Input(UI_element):
                     self.finish_input()
 
 
+    cdef to_unicode(self,obj):
```
Comment (papr) [inline pyglui/ui_elements.pxi]: Exactly what we needed!

**Comment by mkassner** [comment]
basestring in cython contains str, bytes and unicode. str is cython in bytes for py2.7 and unicode for py3 . I have tried to make no assumptions about which of the 3 to use except in text input where we have no good way other than forcing unicode.

**Comment by mkassner** [comment]
ok to merge?

**Pull request merged**